packages/integration-tests/src/integration/create-multinode/1-host-discovery.spec.js

```diff
@@ -71,5 +86,13 @@ describe(`Assisted Installer Multinode Host discovery`, () => {
       });
       commonActions.getNextButton().should('be.enabled');
     });
+
+    describe(`Redesign of host discovery step`, () => {
+      it('Instructions should appear inside the Add Host modal', () => {
```

Comment:
I would combine this in the test where we are checking that the "Generating" button is displayed, to reduce the run time for the tests.